census/core.py

```diff
@@ -518,6 +527,7 @@ def __init__(self, key, year=None, session=None):
         self.acs5 = ACS5Client(key, year, session)
         self.acs3 = ACS3Client(key, year, session)
         self.acs1 = ACS1Client(key, year, session)
+        self.acs5subject = ACS5SubjectClient(key, year, session)
```

Comment:
The naming convention in this library tends to prefer acronyms, e.g. `acs5dp` for data profiles and `acs5sf` for summary files. To match this convention, how about the name `acs5st` for this client?